doc/source/snippets/doc-local-build.rst

```diff
@@ -1,25 +1,61 @@
-#. Install the requirements for building the documentation.
+#. Install documentation related dependencies:
 
     .. code-block:: bash
 
         $ conda install --file requirements/docs.txt
 
-#. Then we will use an external tool called ``sphinx-reload`` to automatically reload the documentation when you make changes to ``.rst`` files.
+#. Enter into the ``doc`` project directory and render documentation:
 
     .. code-block:: bash
 
-        $ pip install sphinx-reload
+        $ cd doc
+        $ make html
 
-    .. note::
+#. Open the rendered documentation via web browser:
 
-        ``sphinx-reload`` is only available via pip install.
+    .. code-block:: bash
+
+        $ open _build/html/index.html
 
-#. A HTML will appear automatically in your browser by running the following command:
+#. Here is a shortcut if you want to use it from the root directory of the project:
 
     .. code-block:: bash
 
-        $ sphinx-reload doc
+        $ cd doc && make html && open _build/html/index.html && cd ..
 
     .. seealso::
```

Comment:
I have added "seealso" where users can use the "shortcut" - this was [@bobleesj](https://github.com/bobleesj)'s way of doing before using sphinx-reload.

<img width="690" alt="Screenshot 2025-05-29 at 9 45 42 AM" src="https://github.com/user-attachments/assets/f93e6df7-0498-473f-aabf-7b9d26b838f6" />